app/renderer/suite.js

```diff
@@ -43,7 +43,7 @@ class Suite {
     }
 
     runAll() {
-        for (const task of this.tasks) {
+        for (let task of this.tasks) {
```

Comment:
Why let here?

Reply:
It is part of `for...of` loop where on every iteration, `task` variable would be updated from one of the value of `this.tasks` list. Although we're not updating `task` inside the loop, it is generally considered good practice to use `let` within `for`.

Reply:
mmm, I always though that using const was better than let, even in these cases, as we are creating a new task variable each iteration.

I don't really mind either way, but I'm not sure on what the good practice is here

Reply:
Generally, if variable will not be modified during the scope of it, `const` is considered ideal, other `let`. And `let` usually goes with `for` as most of the times it gets modified within, however, in this particular case, we can use `const` as well since `task` is never modified within the scope, but having it `let` would be good in case we modify task later. :slightly_smiling_face:

Reply:
Well, my point here is that if we modify the task later, a const will fail, so we simply change to let. While we don't change the variable we could just put const.